### DISCUSSION TITLE
How can I preserve albums as albums?

### DISCUSSION THREAD

**Discussion by ghost** [m4mallu/clonebot#8] [Ideas]
Every time I use this amazing script everything goes well except that albums are forwarded as individual media is there a fix?

**Comment by m4mallu** [comment]
After iteration of messages, look for any media_group present in the message_id. If yes, copy the same.

**Reply by Wileyfox-F1** [reply]
Could you explain further? Didn't get it!

**Reply by m4mallu** [reply]
Presently bot will clone media as individual files, even it's an album. ! If you need to preserve album file files while cloning, edit the code as mentioned. The facility may be added to the clone bot in the future.

**Reply by Wileyfox-F1** [reply]
Oh, Okay! Got it 👍
